.env.example

```diff
@@ -21,17 +22,21 @@ DATABASE_ADAPTER=postgres
 # dev server address:
 CORS_ALLOWED_ORIGINS=http://localhost:8080
 
-# Number of salt rounds to use when hashing user passwords.
+# Auth configuration
 AUTH_SALT_ROUNDS=10
-
-# Session secret string.
+AUTH_JWT_SCHEME=JWT
 AUTH_JWT_SECRET=
+AUTH_JWT_ISSUER=tailor
 
 # Email
-EMAIL_ADDRESS=tailor@extensionengine.com
+EMAIL_SENDER_NAME=Tailor
```

Comment:
Why did we do email service refactor within auth refactor? Did we manage backwards compatibility and set deprecation warnings? This feels like a separate PR 😕 🤔

Reply:
Huh, strictly speaking you are right, it should be separate PR. But if I understand it correctly only breaking change is this `EMAIL_ADDRESS` to `EMAIL_SENDER_*` transition. If so, could we just throw in some deprecation warnings here and proxy old stuff to avoid hassle of factoring those changes out? 🤡

Reply:
👍🏻